docs/topics/request-response.rst

```diff
@@ -316,6 +317,13 @@ download_timeout
 The amount of time (in secs) that the downloader will wait before timing out.
 See also: :setting:`DOWNLOAD_TIMEOUT`.
 
+.. reqmeta:: download_latency
+
+download_latency
+----------------
+
+The amount of time spent to fetch the response, since the request has been
+started.
```

Comment:
I think it worths mentioning that this meta key becomes available after response is downloaded, and that it should be considered read-only. Other meta keys are used for controlling Scrapy behavior, but not this one.